addons/loyalty/models/res_partner.py

```diff
@@ -36,5 +36,9 @@ def action_view_loyalty_cards(self):
         action = self.env['ir.actions.act_window']._for_xml_id('loyalty.loyalty_card_action')
         all_child = self.with_context(active_test=False).search([('id', 'child_of', self.ids)])
         action['domain'] = [('partner_id', 'in', all_child.ids)]
-        action['context'] = {'search_default_active' : True, 'create': False}
+        action['context'] = {
+            'create': False,
+            'search_default_active': True,
```

Comment:
If we add the active filter, shouldn't it be active_test=False by default so you see them all when removing it? From testing it seems removing it doesn't show archived cards

Reply:
same as above
And here `active` is custom filter with some domain